web/src/components/toolbar/ToolbarOption.vue

```diff
@@ -41,13 +41,26 @@ div
         v-radio.wide.mr-0(v-for="m in options",
             :value="m.value", :key="m.value")
           template(#label)
-            span.grow {{ m.label }}
-            help-dialog(v-if="m.helpText", :title="`${m.label} ${title}`",
+            span.grow.groupCombinationContainer(
+                v-bind:title="m.label",
+                style="overflow: hidden") {{ m.label }}
```

Comment:
```suggestion
            span.grow.groupCombinationContainer(v-bind:title="m.label")
              | {{ m.label }}
```
the `style="overflow: hidden"` is redundant with the class.